docs/sphinx_api_docs_source/utils.py

```diff
@@ -73,6 +73,59 @@ def apply_structure_changes(soup, html_file_path, html_file_contents):
         if item.string and ("CodeBlock" not in item.string):
             item.string.replaceWith(item.get_text().replace("<", r"\<"))
 
+    display_methods_details_as_tables(soup)
+
+
+def display_methods_details_as_tables(soup):
+    for item in soup.select(".field-list"):
+        for dd in item.select("dd"):
+            previous_sibling = dd.find_previous_sibling("dt")
+            if previous_sibling is not None:
+                previous_sibling_text = previous_sibling.get_text()
+                if previous_sibling_text in ["Parameters", "Returns", "Raises"]:
+                    create_table(soup, item, dd, previous_sibling_text)
+
+
+def create_table(soup, item, dd, title):
+    table = soup.new_tag("table", attrs={"class": "table"})
+    tbody = soup.new_tag("tbody")
+
+    if title == "Parameters":
+        create_header_row(soup, table, ["Name", "Description"])
+    else:
+        create_header_row(soup, table, ["Type", "Description"])
+
+    p = dd.find("p")
```

Comment:
We should return here if p is not found
```suggestion
    p = dd.find("p")
    if p is None:
        return 
```